library_generation/cli/entry_point.py

```diff
@@ -88,6 +88,20 @@ def generate(
     The commit history, if generated, will be available in
     repository_path/pr_description.txt.
     """
+    generate_impl(
```

Comment:
Why create another wrapper function?

Reply:
Good question. Although the [click documentation](https://click.palletsprojects.com/en/8.1.x/testing/#basic-testing) suggests to use its special `runner`, it would call the production functions `generate_from_yaml` (and others) all the way to produce a full generation, which is a use case I was avoiding.

My approach was to mock the generation functions and only confirm they were called with the expected arguments. However, click throws an error when you try to call a function annotated with `@main.command` directly. That's why I converted the function into a wrapper.